tests/testthat/setup-bugRzilla.R

```diff
@@ -15,7 +16,7 @@ use_key()
 invisible(vcr::vcr_configure(
   dir = vcr_dir,
   filter_request_headers = list(
-      "Authorization" = "My bearer token is safe",
-      "X-BUGZILLA-API-KEY" = "Removing this header too just in case")
+    "Authorization" = "My bearer token is safe",
+    "X-BUGZILLA-API-KEY" = "Removing this header too just in case")
```

Comment:
Note that here there is a change on the indentation, I don't mind but other projects do and might reject everything just for this. (It messes with other people working on this file potentially creating merge conflicts).

Reply:
thanks for notifying me.